api/comments/serializers.py

```diff
@@ -71,7 +75,7 @@ def get_can_edit(self, obj):
         return obj.user._id == user._id
 
     def get_has_children(self, obj):
-        return bool(getattr(obj, 'commented', []))
+        return Comment.find(Q('target', 'eq', obj)).count() > 0
```

Comment:
A MongoDB index. It's a good idea to add `index=True` to fields that you will query often.